### ISSUE TITLE
Meow causes magit to drop stashes instead of discarding changes

### ISSUE THREAD

**Issue by larstvei** [meow-edit/meow#493]
Hi!

Thank you for providing an emacs-friendly introduction to modal editing!

When placing the cursor on a change in the magit status buffer, pressing `k` should drop the change. If the cursor is on a stash, it should drop the stash.

When using meow, i press `SPC k` instead. However, it often disregards the cursor location (which is on a change), and rather goes to drop the stash.

Disabling `meow-global-mode` and pressing `k` allows me to drop the change, rather than the stash.

The bug only occurs sporadically, and so I am not able to reliably reproduce it. Do you have any idea of what could cause the error?

This is my magit configuration:
```elisp
  (use-package magit
    :bind (:map custom-bindings-map ("C-c m" . magit-status)))
```

This is my meow configuration:
```elisp
  (use-package meow
     :config
     (setq meow-cheatsheet-layout meow-cheatsheet-layout-qwerty)
     (add-to-list 'meow-mode-state-list '(vterm-mode . insert))
     (add-to-list 'meow-mode-state-list '(comint-mode . insert))
     (meow-motion-overwrite-define-key
      '("j" . meow-next)
      '("k" . meow-prev)
      '("<escape>" . ignore))
     (meow-leader-define-key
      ;; SPC j/k will run the original command in MOTION state.
      '("j" . "H-j")
      '("k" . "H-k")
      ;; Use SPC (0-9) for digit arguments.
      '("1" . meow-digit-argument)
      '("2" . meow-digit-argument)
      '("3" . meow-digit-argument)
      '("4" . meow-digit-argument)
      '("5" . meow-digit-argument)
      '("6" . meow-digit-argument)
      '("7" . meow-digit-argument)
      '("8" . meow-digit-argument)
      '("9" . meow-digit-argument)
      '("0" . meow-digit-argument)
      '("/" . meow-keypad-describe-key)
      '("?" . meow-cheatsheet))
     (meow-normal-define-key
      '("0" . meow-expand-0)
      '("9" . meow-expand-9)
      '("8" . meow-expand-8)
      '("7" . meow-expand-7)
      '("6" . meow-expand-6)
      '("5" . meow-expand-5)
      '("4" . meow-expand-4)
      '("3" . meow-expand-3)
      '("2" . meow-expand-2)
      '("1" . meow-expand-1)
      '("-" . negative-argument)
      '(";" . meow-reverse)
      '("," . meow-inner-of-thing)
      '("." . meow-bounds-of-thing)
      '("[" . meow-beginning-of-thing)
      '("]" . meow-end-of-thing)
      '("a" . meow-append)
      '("A" . meow-open-below)
      '("b" . meow-back-word)
      '("B" . meow-back-symbol)
      '("c" . meow-change)
      '("d" . meow-delete)
      '("D" . meow-backward-delete)
      '("e" . meow-next-word)
      '("E" . meow-next-symbol)
      '("f" . meow-find)
      '("g" . meow-cancel-selection)
      '("G" . meow-grab)
      '("h" . meow-left)
      '("H" . meow-left-expand)
      '("i" . meow-insert)
      '("I" . meow-open-above)
      '("j" . meow-next)
      '("J" . meow-next-expand)
      '("k" . meow-prev)
      '("K" . meow-prev-expand)
      '("l" . meow-right)
      '("L" . meow-right-expand)
      '("m" . meow-join)
      '("n" . meow-search)
      '("o" . meow-block)
      '("O" . meow-to-block)
      '("p" . meow-yank)
      '("q" . meow-quit)
      '("Q" . meow-goto-line)
      '("r" . meow-replace)
      '("R" . meow-swap-grab)
      '("s" . meow-kill)
      '("t" . meow-till)
      '("u" . meow-undo)
      '("U" . meow-undo-in-selection)
      '("v" . meow-visit)
      '("w" . meow-mark-word)
      '("W" . meow-mark-symbol)
      '("x" . meow-line)
      '("X" . meow-goto-line)
      '("y" . meow-save)
      '("Y" . meow-sync-grab)
      '("z" . meow-pop-selection)
      '("'" . repeat)
      '("<escape>" . ignore))
     (meow-global-mode 1))
```

I apologize for not being able to provide an easily reproducible bug report, so I completely understand if you close the issue.

**Comment by DogLooksGood** [comment]
I guess it's because we rebind the original command to `H-<key>`, however in magit, the binding is position related, that's why it doesn't work. I have no solution for now, the best idea is to not remap in the MOTION state, unless we find a beter way. 

We can keep this issue open.

**Comment by montchr** [comment]
FWIW this also seems to affect the binding for `magit-discard`, which also defaults to `k` in the status screen. I can confirm that pressing `SPC k` works instead.

I also noticed that `describe-bindings` is not matching up with the actual bindings or the defaults in the Magit manual [[link](https://magit.vc/manual/magit/Keystroke-Index.html)]:

```
<1>							magit-apply
<2>							magit-stage
<3>							magit-unstage
<4>							magit-discard
<5>							magit-reverse
<6>							magit-diff-visit-file
<7>							magit-file-untrack
<8>							magit-file-rename
```

**Comment by montchr** [comment]
I've resurrected part of my previous comment because I still think the `describe-bindings` issue may be relevant. (I marked that comment as off-topic shortly after I posted it, and now there doesn't seem to be a way to revert that status)

I feel like it obstructs the discoverability of the workaround -- a solution should be obvious inside Emacs without having to search the issue tracker for an explanation.


I also noticed that `describe-bindings` is not matching up with the actual bindings or the defaults in the Magit manual [[link](https://magit.vc/manual/magit/Keystroke-Index.html)]:

```
<1>							magit-apply
<2>							magit-stage
<3>							magit-unstage
<4>							magit-discard
<5>							magit-reverse
<6>							magit-diff-visit-file
<7>							magit-file-untrack
<8>							magit-file-rename
```

**Comment by DogLooksGood** [comment]
Fixed #517
